client/mysql_upgrade.c

```diff
@@ -1243,7 +1244,12 @@ static int run_sql_fix_privilege_tables(void)
       dynstr_append(&ds_script, *query_ptr);
   }
 
-  run_query(ds_script.str,
+  if (opt_verbose>=5)
+    run_query(ds_script.str,
+            NULL, /* Send output from mysqlcheck directly to screen */
```

Comment:
This comment is wrong.
You are not running `mysqlcheck`, but `mysql` instead and this comment is not needed. 
No reason command to be in multiple lines.

Reply:
could also use a conditional operator here too:
`run_query(ds_script.str, (opt_verbose >= 5) ? NULL : &ds_result. TRUE)`

Reply:
thanks, done.